care/emr/migrations/0031_auto_20250429_1129.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 5.1.4 on 2025-04-29 05:59
+
+from django.db import migrations, transaction
+
+from care.emr.resources.template.spec import FacilityReportTemplateType
+from care.utils.reports.load_default_report_config import load_default_discharge_summary_config
+
+
+def add_default_summary_report_config_to_facility(apps, schema_editor):
+    Facility = apps.get_model("facility", "Facility")
+    FacilityReportTemplate = apps.get_model("emr", "FacilityReportTemplate")
+
+    with transaction.atomic():
+        for facility in Facility.objects.all():
+            FacilityReportTemplate.objects.create(facility=facility,type=FacilityReportTemplateType.discharge_summary,config=load_default_discharge_summary_config())
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('emr', '0030_facilityreporttemplate'),
+    ]
+
+    operations = [
+        migrations.RunPython(add_default_summary_report_config_to_facility),
```

Comment:
```suggestion
        migrations.RunPython(add_default_summary_report_config_to_facility, migrations.RunPython.noop),
```
allow migrations to be reversible